custom-namespaces.yaml

```diff
@@ -599,10 +599,62 @@ firefox_accounts:
       type: table_view
       tables:
         - table: moz-fx-data-shared-prod.accounts_frontend_derived.login_engagement_funnel_v1
+    health_cost:
+      type: table_view
+      tables:
+        - table: mozdata.analysis.wclouser_fxa_health_cost
+      measures:
+        count:
+          type: count
+    health_counts:
+      type: table_view
+      tables:
+        - table: mozdata.analysis.wclouser_fxa_health_counts
+      measures:
+        count:
+          type: count
+    health_db_counts:
+      type: table_view
+      tables:
+        - table: mozdata.analysis.wclouser_fxa_health_db_counts
+      measures:
+        count:
```

Comment:
This is causing a LookML error because the underlying table already has a `count` column which gets a `count` Looker dimension auto-generated for it, so you can't also add a Looker measure named `count`:
> Redefinition of field 'health_db_counts.count'
[Learn more.](https://cloud.google.com/looker/docs/r/kb/analyst/redefinition-of-field-error?version=24.18)
[`health_db_counts.view:33 firefox_accounts:health_db_counts`](https://mozilla.cloud.looker.com/projects/looker-hub/files/firefox_accounts/views/health_db_counts.view.lkml?line=33)